nuxeo-webdav/src/main/java/org/nuxeo/ecm/webdav/backend/SimpleBackend.java

```diff
@@ -72,8 +70,6 @@
 
     protected TrashService trashService;
 
-    protected PathCache pathCache;
```

Comment:
It would have been nice to do the pathCache removal in a separate commit for clarity, as this is not directly tied to the MS Office work.

Reply:
Done!